docs/apis/storm_compatibility.md

```diff
@@ -201,6 +201,26 @@ DataStream<BoltOutputType> s2 = splitStream.select("s2").transform(/* use Bolt f
 
 See [SpoutSplitExample.java](https://github.com/apache/flink/tree/master/flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/split/SpoutSplitExample.java) for a full example.
 
+## Configure for embedded Spouts/Bolts
+Embedder spouts/bolts can be configure with user defined parameters. User defined parameters is a config `Map` and be called by `Spout.open(...)` and `Bolt.prepare()`
```

Comment:
Typo: Embedded
Please spell Spouts and Bolts always with a capital letter (for consistency over the whole documentation)
Please also follow a "one sentence per line" source code style. It makes diffs easier to read in the future.